packages/token/src/erc20/interface.cairo

```diff
@@ -3,73 +3,164 @@
 
 use starknet::ContractAddress;
 
+/// Interface of the ERC20 standard as defined in the EIP.
```

Comment:
For consistency, let's remove the comment in the interface, and keep only the ones per function.

Reply:
Except in camelCase where it should be added just to the interface as mentioned in another comment.